designs/developer/helm-45/helm-notes.md

```diff
@@ -0,0 +1,14 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Helm Release Notes
+After successful installation of a helm chart, users are directed to the topology view where they could see it visualized along with its components. On the topology view, the helm chart comes selected by default with the side panel showing release notes(from the associated Notes.txt file) for the respective helm chart.
+
+![Helm release notes on topology](img/helm-release-side-panel.png)
+
+
+Similarly, a release notes tab would also be introduced in the details page of the VM based resource. In the future users can access the notes for a given helm chart through the side-panel or the details page, under the release notes tab.
```

Comment:
should "Release Notes" have initial caps?

Reply:
"Users can access the notes for a given helm chart after installation..." might be better

Reply:
Changed